web-ui/src/main/resources/catalog/components/metadataactions/RelatedResourcesService.js

```diff
@@ -119,62 +119,7 @@
           };
 
 
-          var addWMTSToMap = function(link, md) {
```

Comment:
where did this code go ?

Reply:
It is here gnViewerSettings.resultviewFns.addMdLayerToMap;

We used to have 2 ways of adding layer to maps - one for search results, one for the related directive. I think it is good to have only one entry point for doing this. Having it in resultviewFns make it also easy to customize if needed.

Reply:
> It is here gnViewerSettings.resultviewFns.addMdLayerToMap;

Yes i saw that, but where is the code
I would rather have made `gnViewerSettings.resultviewFns.addMdLayerToMap` pointing on `mapService` method.